traffic_ops/traffic_ops_golang/urisigning/urisigning.go

```diff
@@ -63,7 +70,8 @@ func GetURIsignkeysHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	if len(ro) == 0 {
-		ro, err = json.Marshal(jwk.NewSet())
+		ro, err = json.Marshal(emptyURISigningKey)
+		panic(ro)
 		if err != nil {
```

Comment:
The `go vet` GitHub Action fails here:

> ```go
> # github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/urisigning
> traffic_ops/traffic_ops_golang/urisigning/urisigning.go:75:3: unreachable code
> ```

Reply:
Argh, that was a debug statement. Sorry, I was looking for how to reproduce that path in the tests. I'll remove it now

Reply:
should be fixed in 2ca737ccb